dotCMS/src/main/java/com/dotmarketing/common/db/DotConnect.java

```diff
@@ -605,6 +606,11 @@ private void executeQuery(Connection conn) throws SQLException {
                 }
 
                 // statement.setMaxRows(maxRows);
+                if (this.fetchSize > 0) {
+
+                    statement.setFetchSize(this.fetchSize);
```

Comment:
This code does not work as expected - regardless of the fetch size, we read all of the results into memory.  We should use raw jdbc, and not use DotConnect  when fetching the file.

Reply:
done